### DISCUSSION TITLE
Has anyone noticed that Demo of this project couldn't access it?

### DISCUSSION THREAD

**Discussion by zkeq** [Ashutosh00710/github-readme-activity-graph#117] [Q&A] [answered]
I found that I have encountered such a problem.

Where is this failure?

![2](https://user-images.githubusercontent.com/62864752/164967296-ce257ac3-a188-4f11-914b-83b86e8292d4.png)

**Comment by Ashutosh00710** [comment, marked as answer]
This error is due to #55. But now service is up and running 🚀

**Comment by AftabNaiyer** [comment]
This error is due to server down
